packages/server/modules/core/services/streams.js

```diff
@@ -52,14 +50,6 @@ module.exports = {
     return await revokeStreamPermissions({ streamId, userId })
   },
 
-  /**
-   * @deprecated Use deleteStreamAndNotify or use the repository function directly
-   */
-  async deleteStream({ streamId }) {
```

Comment:
Deleted cause its duplicate. The only real difference is the extra log msg, but removing it is fine cause 1) its just spam 2) this pretty much only gets invoked in tests.